go.mod

```diff
@@ -188,7 +188,7 @@ require (
 	golang.org/x/tools v0.16.1 // indirect
 	google.golang.org/appengine v1.6.8 // indirect
 	google.golang.org/protobuf v1.31.0 // indirect
-	gopkg.in/go-jose/go-jose.v2 v2.6.1 // indirect
+	gopkg.in/go-jose/go-jose.v2 v2.6.3 // indirect
```

Comment:
Pinning a specific dep is better since you pull minimal changes in. Also, you are guaranteed the version does not change. On the other hand, if there's a reliable upstream dep that can pull the same go-jose version, let's go with the upstream dep. E.g. k8s. If not (e.g. in case the upstream dep changes go-jose version back), let's keep pining go-jose directly.